web_external/Studies/studyPage.pug

```diff
@@ -41,12 +41,25 @@
                 td.isic-study-user-name= user.name()
                 td
                   if study.canAdmin()
-                    a.isic-study-remove-user-button(title='Delete annotator')
+                    a.isic-study-remove-user-button.isic-study-delete-button(title='Delete annotator')
                       i.icon-cancel
                 td.isic-study-user-completion-count= numCompleted
                 td.isic-study-user-completion-percent= percentCompleted
         else
           | (no users)
+      if study.canAdmin() && study.get('waitingUsers').length
```

Comment:
Can we always show this field to admins, even if there are currently no waiting users (to train admins to keep in mind that this functionality exists)?

Maybe show a fallback placeholder if the list is empty.